.github/workflows/service-worker-test.yml

```
@@ -0,0 +1,33 @@
+name: Test the service worker automatic update between two version
+
+on:
+  workflow_dispatch:
+    inputs:
+      versionFrom:
+        description: "Nom de la release (ex: 199.0.0)"
+        required: true
+        type: string
+      versionTo:
+        description: "Nom de la release (ex: 199.0.0)"
```

```suggestion
        description: "Nom de la release (ex: 200.0.0)"
```
Pour voir rapidement qu'il en faut 2 différentes et laquelle est la plus avancée

---

C'est une bonne remarque, j'ai fais cela car j'avais besoin d'un script en local pour tester ma solution sans CI, je peux refacto cela quand j'ai de quoi gérer la CI, que je découvre aussi GH action.